scripts/generate-integration-pages.ts

```diff
@@ -109,6 +115,8 @@ class IntegrationPagesBuilder {
 
 layout: ~/layouts/IntegrationLayout.astro
 title: '${name}'
+description: ${createDescription(name,category)}
+installCMD: npx astro add ${name.replace('@astrojs/','')}
```

Comment:
aye this was because with the way the canvas is setup you cant pass through `\n` this was done so the install command could be called to a second line, and I figured that it would make sense to have this included within the frontmatter. 
But given that this kinda escaped the scope of the pr Ill make the changes.